tech-specs/src/explore/design/dkg.md

```diff
@@ -1 +1,154 @@
-# Distributed key generation gossip
+# Distributed key generation
+
+Here we give an overview of how the DKG protocol works with the ledger and 
+the roles of each node in the network per this protocol. This portion of the 
+book focuses solely on the production of a new public encryption key via the
+DKG scheme. It does not cover encryption, decryption, or validation of transactions.
+
+Some parts of the DKG protocol may be done through the gossip layer or through
+off-chain communication. However, the entire DKG cannot exist solely in the 
+gossip layer as consensus on the final shared secrets must be achieved. This 
+will require posting information on chain.
+
+## Starting the Protocol
+
+To initializing the protocol to generate a new key, the  current block 
+proposer proposes a new instance. This would be posted on the blockchain and
+notify all the participating validators to begin the announcement phase. 
+However, when should the proposer make this decision? In all likelihood it will be a combination of blocks transpired since
```

Comment:
bad phrasing, the proposer doesn't make this decision, this is determined by the protocol (prior to the start of a new epoch)

cc @joebebel

Reply:
Right, it's best if the protocol explicitly specifies the height, then there's no need for an explicit announcement

Reply:
:+1:

Reply:
I assume that we will initialize the protocol at fixed blockheights with an exception being that if the protocol fails, we immediately start again?